        void accept(String name, Object value) throws SAXException;

